app/assets/stylesheets/application.scss

```diff
@@ -282,23 +290,23 @@ table {
   }
 
   .sorting {
-    background: image-url('sort_both.png') no-repeat center right;
+    background: url('sort_both.png') no-repeat center right;
```

Comment:
this is actually used by data tables (for example, in compute resource->show->virtual machines tab).